### ISSUE TITLE
[d3d11] Battlefield 4 crashes on start - "Memory could not be read"

### ISSUE THREAD

**Issue by sKGu8gxV** [doitsujin/dxvk#1657]
First off thanks for the amazing work you are doing with this!

Battlefield 4 crashes on start in both Windows 8.1 and Windows 10 LTSB 2016 (Fully upgraded) with error message: 
"The instructions at 0x0007FFBE18B7CA referenced memory at 0x00000000000000. The memory could not be read."

### System information
- GPU: Geforce 1080
- Driver: 442.50 in both 8.1/10
- DXVK version: Tested with 1.7 and 1.61

I also reset the settings, so they don't seem to be to blame (renamed the Battlefield 4 settings folder).

### Log files
- bf4_d3d11.log is empty
- bf4_dxgi.log: https://pastebin.com/cswRnXX0

Not sure how to create an apitrace on windows tbh.

**Comment by doitsujin** [comment]
BF4 works on Linux, I have no plans to look into Windows-specific crashes since they are almost never actual DXVK bugs anyway.

**Comment by Thinkaboutmin** [comment]
I did not have this problem when I was testing BF4 on Windows, it worked flawless. I was using the latest Windows 10 Pro version.

Just in case, are you using 32bit dxvk? Try putting the 64bit dlls onto the game folder, it _may_ solve the issue.

**Comment by sKGu8gxV** [comment]
I tested both the 32 and 64 bit binaries since Bf4 comes with both. Got the same error with both. I also tested dxvk with both Mpv and Mpv.net (Which supposedly is just a gui around Mpv.) since then. It works with Mpv if I start it with Nsudo (gives trustedinstaller rights) or before I install Directx on a freshly installed, very stripped, windows 10. It also seems to work with Mpv.net in "Vanilla". It doesn't seem I can run Bf4 withouth Origin running the Dxsetup installer unfortunately after a fresh install.
